packages/ui/components/test-setup.tsx

```diff
@@ -53,6 +55,9 @@ vi.mock("next/navigation", async () => ({
   useSearchParams() {
     return new URLSearchParams();
   },
+  usePathname() {
+    return "/settings/billing";
+  },
```

Comment:
Doesn't make sense to hardcode to settings/billing in test-setup.tsx which is general mocking